tests/unit_tests/kd_model_test.py

```diff
@@ -205,6 +206,21 @@ def test_train_kd_module_pretrained_ckpt(self):
                              )
         sg_model.train(self.kd_train_params)
 
+    def test_build_model_with_input_adapter(self):
+        kd_model = KDModel("build_kd_module_with_with_input_adapter", device='cpu')
+        kd_model.build_model(student_architecture='resnet18', teacher_architecture='resnet50',
+                             student_arch_params={'num_classes': 1000}, teacher_arch_params={'num_classes': 1000},
+                             checkpoint_params={'teacher_pretrained_weights': "imagenet"},
+                             arch_params={
+                                 "teacher_input_adapter": NormalizationAdapter(mean_original=[0.485, 0.456, 0.406],
+                                                                               std_original=[0.229, 0.224, 0.225],
+                                                                               mean_required=[0.5, 0.5, 0.5],
+                                                                               std_required=[0.5, 0.5, 0.5])})
+        imagenet_resnet50_sg_model = SgModel("pretrained_resnet50")
```

Comment:
@shaydeci  - What are lines 219-221 for?

Reply:
fixed